tests/start_proxy/start_proxy_test.py

```diff
@@ -19,7 +19,7 @@
 
 currentdir = os.path.dirname(
     os.path.abspath(inspect.getfile(inspect.currentframe())))
-sys.path.append(currentdir + "/../../docker/generic")
+sys.path.insert(0, currentdir + "/../../docker/generic")
```

Comment:
I couldn't get the tests to run on my machine, SO suggests to replace the first path instead of appending. This works now, should work in CI.